### PR TITLE
common: fix access and add name for the token bucket throttle.

### PR DESCRIPTION
**Pull request by iridescent-rsy**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/common/Throttle.h [hunk 1]
```diff
@@ -357,6 +357,7 @@ class TokenBucketThrottle {
   };
 
   CephContext *m_cct;
+  const std::string m_name;
```
[the remark this answers is not in the document]
Comment (iridescent-rsy) [reply src/common/Throttle.h]: @yangdongsheng May I remove the `m_` of all members in  this class? As I seen, most classes do not have `m_` for their members.

[the remark this answers is not in the document]
Comment (yangdongsheng) [reply src/common/Throttle.h]: what's your opinion @dillaman ?

[the remark this answers is not in the document]
Comment (dillaman) [reply src/common/Throttle.h]: I would just do something like:

```
#undef dout_prefix
#define dout_prefix *_cout << "TokenBucketThrottle(" << m_name << " ...
```

**Comment by yangdongsheng** [comment]
agreed, just redefine the dout_prefix with m_name before 
TokenBucketThrottle.

**Comment by yangdongsheng** [comment]
@dillaman  please take a look

**Pull request merged**